pkg/runner/local_docker.go

```diff
@@ -64,6 +66,7 @@ var defaultConfig = LocalDockerRunnerConfig{
 	KeepContainers: false,
 	Unstarted:      false,
 	Background:     false,
+	Ulimits:        []string{""},
```

Comment:
The intention was to set a high default limit (1Mi), as suggested in https://github.com/ipfs/testground/issues/781.